apps/gallery/js/startup.js

```diff
@@ -93,6 +93,11 @@
       // If we were viewing or editing an image it might not be there
       // anymore when the MediaDB becomes available again.
       if (!picking) {
+        // If we are editing image, call exitEdit to revoke blob URLs
+        // and close the editor object.
```

Comment:
Please add a note indicating that this is a known case where the user can lose their work.